.cloud-build/execute_changed_notebooks_cli.py

```diff
@@ -129,26 +139,65 @@ def str2bool(v):
     base_branch=args.base_branch,
 )
 
+
+def _load_results() -> list:
+    '''
+    Load accumulated notebook test results
+    '''
+    print("Loading existing accumulative results ...")
+    rows = []
+    try:
+        df = pd.read_csv(os.path.join(f"gs://{args.artifacts_bucket}", args.test_results))
+        df = df.reset_index()
+
+        for index, row in df.iterrows():
+            rows.append([row["notebook"], row["duration"], row["passed"], row["failed"]])
```

Comment:
Use a dict instead.

In fact you can replace the entire loop with:

`df[["notebook", "duration", "passed", "failed"]].to_dict('records')`

Reply:
done

Reply:
I don't see any changes.